### PR TITLE
#305 removed the fixture

### PR DESCRIPTION
**Pull request by ammaratef45**
PR for #305 
Removed the fixture and tests are fine

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#332](https://github.com/timelesslounge/timelessis/issues/332) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#332](https://github.com/timelesslounge/timelessis/issues/332) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/9cbf68c5-b152-4817-b10c-113ed0fcb8df); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by vladarefiev** [comment]
@rultor merge. nothing to check

**Comment by vladarefiev** [comment]
@emilianodellacasa wait pls

File: tests/conftest.py [hunk 1]
```diff
@@ -57,9 +57,3 @@ def _db(app):
     Flask-SQLAlchemy database connection.
     """
     return DB
-
-
-@pytest.fixture
-def clear_cache(app):
```
Comment (vladarefiev) [inline tests/conftest.py]: @ammaratef45 I dont see a way to delete it. We definitely should clear cache before every test

File: tests/conftest.py [hunk 2]
```diff
@@ -57,9 +57,3 @@ def _db(app):
     Flask-SQLAlchemy database connection.
     """
     return DB
-
-
-@pytest.fixture
```
Comment (vladarefiev) [inline tests/conftest.py]: @ammaratef45 Even say more we should make it `autouse=True`

File: tests/conftest.py (hunk 1 above)
Comment (ammaratef45) [reply tests/conftest.py]: @vladarefiev that was the puzzle #305 
I may clear the cache in the app fixture
What do you think?

Comment (vladarefiev) [reply tests/conftest.py]: @ammaratef45 I think no need to make one fixture more complexity.
I prefer leave this one with `autouse=True` may be

**Comment by ammaratef45** [comment]
@vladarefiev what do you think after deefb8a?

**Comment by vladarefiev** [comment]
@ammaratef45 good to me, thanks

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 3 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#332](https://github.com/timelesslounge/timelessis/issues/332) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vladarefiev** [comment]
@ypshenychka I thought two, but it may be one, confirm.

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**